### PR TITLE
Add max frame rate slider

### PR DESCRIPTION
**Pull request by KaptenJansson**
**Description**
Add max frame rate slider 

**Purpose**
Should be able to constrain the max frame rate.

### CODE REVIEW

**Comment by KaptenJansson** [comment]
Ping

**Comment by alvestrand** [comment]
lgtm
thought for future revision of https://webrtc.github.io/samples/src/content/peerconnection/constraints/ - is it possible (css???) to push the min and max sliders for the same property closer together, so that they are visually paired?

the list of sliders is getting long.

File: src/content/peerconnection/constraints/index.html [hunk 1]
```diff
@@ -69,8 +69,12 @@
...
             <input type="range" min="0" max="1080" value="480">
           </div>
-          <div id="framerate">
-            <label>Frame rate <span>0</span>fps:</label>
+          <div id="minFramerate">
+            <label>Min Frame rate <span>0</span>fps:</label>
+            <input type="range" min="0" max="60" value="0">
+          </div>
+          <div id="maxFramerate">
+            <label>Max Frame rate <span>0</span>fps:</label>
```
Comment (samdutton) [inline src/content/peerconnection/constraints/index.html]: Maybe change this to **Min frameRate** and  **Max frameRate**. 

This has the handy side effect that the label fits!

Nitpick — also noticed that the CSS could do with a little tweaking: remove the border-bottom from div#getUserMedia and make the div#input right margin 4px: `margin: 0 4px 0 0;`

**Comment by samdutton** [comment]
LGTM % nitpick, with merge. (Apologies for delay in getting onto this!)

File: src/content/peerconnection/constraints/index.html (hunk 1 above)
Comment (samdutton) [reply src/content/peerconnection/constraints/index.html]: Ping @KaptenJansson — do you want to do these little tweaks?

**Comment by alvestrand** [comment]
@samdutton googlebot seems confused by your merge commit. Do you know how to fix that?

**Comment by fippo** [comment]
git rebase -i gh-pages
and then force push might help?

**Comment by alvestrand** [comment]
lgtm

**Comment by KaptenJansson** [comment]
I've never gotten the googlebot to change the CLA label back to yes. Not sure if a comment from @samdutton will solve it. I've manually changed it to yes before though.

**Comment by samdutton** [comment]
Attempt to appease googlebot :).

**Pull request merged**